From: Jan Beich <jbeich@FreeBSD.org>
Date: Wed, 29 Sep 2021 13:12:33 +0000
Subject: meson: make posix_fallocate visible in glibc with -std=c99

meson.build:59:4: ERROR: Problem encountered: Required function posix_fallocate missing

Origin: upstream, 0.1.1, commit:8ee7ebe87ea76de3854827b7b0666f78bd84d416
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index ad4ccb9..4b50935 100644
--- a/meson.build
+++ b/meson.build
@@ -51,7 +51,7 @@ dbus_dep = dependency('dbus-1', version: dbus_req, required: get_option('dbus'))
 # list of (function, prefix)
 required_functions = [
   ['memfd_create', '#define _GNU_SOURCE\n#include <sys/mman.h>'],
-  ['posix_fallocate', '#include <fcntl.h>'],
+  ['posix_fallocate', '#define _GNU_SOURCE\n#include <fcntl.h>'],
   ['asprintf', '#define _GNU_SOURCE\n#include <stdio.h>'],
 ]
 foreach function : required_functions
